test/utils.js

```diff
@@ -34,14 +35,28 @@ describe('Operatable', () => {
     });
 
     it('Should not be able to run the onlyOperatorable function as not the operator', async () => {
-        await expect(test.connect(otherWallet).testOperatorable()).to.be.revertedWithCustomError(test, 'NotOperator');
+        await expectRevert((gasOptions) => test.connect(otherWallet).testOperatorable(gasOptions), test, 'NotOperator');
     });
 
     it('Should be able to change the operator only as the operator', async () => {
         expect(await test.operator()).to.equal(ownerWallet.address);
         await expect(test.transferOperatorship(otherWallet.address)).to.emit(test, 'OperatorshipTransferred').withArgs(otherWallet.address);
         expect(await test.operator()).to.equal(otherWallet.address);
-        await expect(test.transferOperatorship(otherWallet.address)).to.be.revertedWithCustomError(test, 'NotOperator');
+        await expectRevert((gasOptions) => test.transferOperatorship(otherWallet.address, gasOptions), test, 'NotOperator');
+    });
+
+    it('Should be able to propose operator only as the operator', async () => {
+        expect(await test.operator()).to.equal(otherWallet.address);
+        await expectRevert((gasOptions) => test.proposeOperatorship(ownerWallet.address, gasOptions), test, 'NotOperator');
+        await expect(test.connect(otherWallet).proposeOperatorship(ownerWallet.address))
```

Comment:
newline between statements/awaits would be good

Reply:
added additional spacing for readability